repos/fdbt-site/tests/pages/products/productDetails.test.tsx

```diff
@@ -196,6 +196,7 @@ describe('myfares pages', () => {
                     lineId: 'q2gv2ve',
                     passengerTypeId: 9,
                     csrfToken: '',
+                    fareTriangleModified: undefined,
```

Comment:
Should we add a render test, and a props test, for when fareTriangleModified isnt undefined?

Reply:
done